src/app/main-page/main-page.controller.js

```diff
@@ -21,8 +24,65 @@ export default class MainPageController {
             .catch(response => {
                 console.log('There are no nearby cities');
             });
-        console.log('Cities: ', this.nearbyCities);
+        this.openWeatherMapsService.getCurrentWeather(this.param)
+            .then(response => {
```

Comment:
maybe it's a good idea to move this logic to an utils or service, what do you think??